refarch-backend/src/main/resources/application-local.yml

```diff
@@ -22,16 +22,17 @@ spring:
     oauth2:
       resourceserver:
         jwt:
-          jwk-set-uri: ${keycloak.auth-server-url}/auth/realms/${realm}/protocol/openid-connect/certs
+          jwk-set-uri: ${sso.url}/auth/realms/${sso.realm}/protocol/openid-connect/certs
           audiences:
-            - local
+            - ${sso.client}
 
-# Define the local keycloak realm here
-realm: local_realm
-keycloak:
-  auth-server-url: http://keycloak:8100
+# Define the local keycloak configuration here
+sso:
+  url: http://localhost:8100
+  realm: local_realm
+  client: local
```

Comment:
I like that you grouped those attributes together. But should we place it under `refarch` key? I know those are not properties we use inside the code and are just placed here to avoid duplication but still.

Reply:
i dont really see a benefit by doing that. I would prefer how it is currently done